### PR TITLE
Project patern

### PR DESCRIPTION
**Pull request by Nigeriobike**
ProjectStart

### CODE REVIEW

**Review by CyberShval — APPROVED** [review]
test

**Review by llo3iXyH — APPROVED** [review]

**Review by CortUA — APPROVED** [review]

**Review by CortUA — APPROVED** [review]

File: src/test/java/TetstNG/TestInit.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
+package TetstNG;
```
Comment (CortUA) [inline src/test/java/TetstNG/TestInit.java:1]: TetstNG - Spelling mistake

**Review by CortUA — APPROVED** [review]

**Comment by DereviankinNikita** [comment]
it would be cool to give some useful names to your commits

**Comment by DereviankinNikita** [comment]
Please provide .gitignore file to exclude unnecessary files and directories from git tracking

**Comment by DereviankinNikita** [comment]
Please provide .gitignore file to exclude unnecessary files and directories from git tracking

**Pull request merged**